APM > .NET > Update setup instructions (#5581)
DataDog/documentation: content/en/tracing/setup/dotnet.md

COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
```

`COR_PROFILER_PATH` is not required because the MSI installer registers the native COM library's path in the Windows Registry, and environment variables `DD_INTEGRATIONS` and `DD_DOTNET_TRACER_HOME` are set globally for all processes.

> If you did not use the MSI installer, set all five environment variables:

Reviewer: ```suggestion
If you did not use the MSI installer, set the following environment variables:
```

Reviewer 2: I think @zacharycmontoya's intent was to emphasize that in this specific case, they need to set _all five_ env vars instead of just two like mentioned a few paragraphs above.

Reviewer: Ok. We can skip changing this if it's important to specify. I usually try to eliminate saying "follow these # of steps" because when things get added or removed, it's easy to forget to update the number.

Author: Yes, for the two lines starting with `If you did not use the MSI installer`, I would like to keep the specific number included to avoid any ambiguity. I am happy to change it for the third suggestion on the Linux setup.

```
COR_ENABLE_PROFILING=1
COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
COR_PROFILER_PATH=%PROGRAMFILES%\Datadog\.NET Tracer\Datadog.Trace.ClrProfiler.Native.dll
DD_INTEGRATIONS=%PROGRAMFILES%\Datadog\.NET Tracer\integrations.json
DD_DOTNET_TRACER_HOME=%PROGRAMFILES%\Datadog\.NET Tracer
```

> For example, to set them from a batch file before starting your application:

Reviewer: ```suggestion
For example, to set the environment variables from a batch file before starting your application:
```

CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
```

> `CORECLR_PROFILER_PATH` is not required because the MSI installer registers the native COM library's path in the Windows Registry, and environment variables `DD_INTEGRATIONS` and `DD_DOTNET_TRACER_HOME` set globally for all processes.

Reviewer: ```suggestion
`CORECLR_PROFILER_PATH` is not required because the MSI installer registers the native COM library's path in the Windows Registry, and environment variables `DD_INTEGRATIONS` and `DD_DOTNET_TRACER_HOME` are set globally for all processes.
```

> If you did not use the MSI installer, set all five environment variables:

Reviewer: ```suggestion
If you did not use the MSI installer, set the following environment variables:
```

Author: As noted above, I'd like to keep this specific number in the documentation to avoid any confusion after reading about the previous two environment variables.

{{% tab ".NET Core on Linux" %}}

> On Linux, these five environment variables are required to enable automatic instrumentation:

Reviewer: ```suggestion
On Linux, the following environment variables are required to enable automatic instrumentation:
```
